src/main/java/org/jenkinsci/test/acceptance/plugins/workflow_multibranch/GithubBranchSource.java

```diff
@@ -15,14 +16,18 @@
 @Describable("GitHub")
 public class GithubBranchSource extends BranchSource {
 
-    public final Control owner = control("source/repoOwner");
-    public final Control repository = control("source/repository");
-    public final Control credential = control("source/scanCredentialsId");
+    public final Control owner = control(by.name("_.repoOwner"));
+    public final Control repository = control(by.name("_.repository"));
+    public final Control credential = control(by.name("_.scanCredentialsId"));
```

Comment:
Note that the `By` form performs page object wide search while the `path` form search with the page area. In case the PA is guaranteed not to appear on page more than once, it is ok.